integration/src/main/java/scripts/testscripts/EnablePet.java

```diff
@@ -60,7 +62,7 @@ protected void doUserJourney(TestUserSpecification testUser, WorkspaceApi userWo
         ClientTestUtils.getWorkspaceClientFromToken(petSaToken, server);
     String petEnableResult = petSaWorkspaceApi.enablePet(getWorkspaceId());
     assertEquals(petSaEmail, petEnableResult);
-    Iam petIamClient = ClientTestUtils.getGcpIamClientFromToken(petSaToken);
+    Iam unused = ClientTestUtils.getGcpIamClientFromToken(petSaToken);
```

Comment:
nit: this is unused because the assertion that would use it is commented out below. Instead of renaming, let's just comment this out as well.